src/modules/solution/type/multiple_choice_solution.py

```diff
@@ -0,0 +1,12 @@
+from .abstract_solution import AbstractSolution
+
+
+class MultipleChoiceSolution(AbstractSolution):
+    def __init__(self, solution: list[str]):
+        super().__init__(sorted(solution))
+
+    def verify(self, solution):
+        return isinstance(solution, MultipleChoiceSolution) and self.get_solution() == solution.get_solution()
+
+    def serialize(self):
+        return str(self.get_solution())
```

Comment:
Consider using `pickle.dump()` from `pickle` library when serializing objects and its counterpart is `pickle.load()`.